jsinterop_generator.bzl

```diff
@@ -260,6 +260,8 @@ def jsinterop_generator(
     visibility = None,
     testonly = None,
     j2cl_test_externs_list = [],
+    gwt_library_tags = [],
+#    j2cl_library_tags = [],
```

Comment:
What do you try to do here?

Reply:
google_bazel_common can generate POM files, and will use specifically formatted tags on java_library dependencies to build the POM `<dependencies>` so we need a way to pass those tags.

Reply:
Ok, this is used in https://github.com/google/elemental2/pull/50 I missed that part sorry.
Did you try to create a java_library that reexport the generated one and add the tags field?

```
jsinterop_generator(
     name = "core",
    ...
)

java_library(
  name = "core_for_pom",
  exports = [":core"],
  tags =  ["maven_coordinates=com.google.elemental2:elemental2-core:ELEMENTAL2_VERSION"]
)

load("@google_bazel_common//tools/maven:pom_file.bzl", "pom_file")

pom_file(
    name = "pom.xml",
    preferred_group_ids = [
        "com.google.elemental2",
        "com.google.jsinterop",
    ],
    targets = [":core_for_pom"],
    template_file = "pom-template.xml",
)
```